src/redist/targets/Branding.targets

```diff
@@ -6,6 +6,7 @@
       <MSBuildExtensionsBrandName>.NET Standard Support for Visual Studio 2015</MSBuildExtensionsBrandName>
       <!-- TODO: "Preview" shouldn't be hardcoded in these properties -->
       <SharedFrameworkBrandName>Microsoft .NET Core Runtime $(CliVersionNoSuffix) - Preview</SharedFrameworkBrandName>
+      <NetCoreAppTargetingPackBrandName>Microsoft .NET Core Targeting Pack $(CliVersionNoSuffix) - Preview</NetCoreAppTargetingPackBrandName>
```

Comment:
It's unclear to me why CLI/SDK versions are used here where I'd expect Core-Setup versioning, and where these versions show up... it seems to be how it's been done before but it's confusing.

Reply:
These branding/versioning strings show up in the SDK installer; see: **src/redist/targets/packaging/osx/clisdk/Distribution-Template**

I believe these are displayed in the SDK installer interface.

`<choice id="{NetCoreAppTargetingPackComponentId}.pkg" visible="true" title="{NetCoreAppTargetingPackBrandName} (x64)" description="The .NET Core Targeting Pack">`

Reply:
I looked into this because I was wondering why I'd never seen any choices before, it turns out this `customize="never"` turns off the choices being displayed in the UI (which I think is a good thing):

https://github.com/dotnet/core-sdk/blob/2086afd1388a35cef833c6e7b110aa07036a1dab/src/redist/targets/packaging/osx/clisdk/Distribution-Template#L5

You can find them with `installer -showChoicesXml -pkg dotnet-sdk-latest-osx-x64.pkg` or manually opening up the pkg and looking at `Distribution`, but otherwise they don't show up.

So, seems fine to me as-is.

Reply:
So what does the UI show? [I setting up my MAC now]

Reply:
<details><summary>Screenshots (click me)</summary>

<img width="777" alt="Screen Shot 2019-04-01 at 5 31 07 PM" src="https://user-images.githubusercontent.com/12819531/55363877-3d389980-54a4-11e9-9370-0fbb85b86103.png">

<img width="732" alt="Screen Shot 2019-04-01 at 5 31 15 PM" src="https://user-images.githubusercontent.com/12819531/55363883-432e7a80-54a4-11e9-97ff-3323d78d14b7.png">

<img width="732" alt="Screen Shot 2019-04-01 at 5 31 26 PM" src="https://user-images.githubusercontent.com/12819531/55363890-47f32e80-54a4-11e9-8077-e712c13251f2.png">

<img width="813" alt="Screen Shot 2019-04-01 at 5 31 38 PM" src="https://user-images.githubusercontent.com/12819531/55363896-4aee1f00-54a4-11e9-8614-431b1b888769.png">

<img width="813" alt="Screen Shot 2019-04-01 at 5 31 40 PM" src="https://user-images.githubusercontent.com/12819531/55363900-4f1a3c80-54a4-11e9-9163-e0ab99833a67.png">

</screenshots>

Reply:
Updated the version of .Net core components to the version that is packaged in the native SDK MacOS installer - PKG